tools/dds/dds-server/lrs-device-controller.cpp

```diff
@@ -378,28 +449,58 @@ lrs_device_controller::lrs_device_controller( rs2::device dev, std::shared_ptr<
         throw std::runtime_error( "Empty dds_device_server" );
 
     _dds_device_server->on_open_streams( [&]( const json & msg ) { start_streaming( msg ); } );
-    _dds_device_server->on_close_streams( [&]( const json & msg ) { stop_streaming( msg ); } );
     _dds_device_server->on_set_option( [&]( const std::shared_ptr< realdds::dds_option > & option, float value ) {
         set_option( option, value );
     } );
     _dds_device_server->on_query_option( [&]( const std::shared_ptr< realdds::dds_option > & option ) -> float {
         return query_option( option );
     } );
 
-    //query_sensors returns a copy of the sensors, we keep it to use same copy throughout the run time.
-    //Otherwise problems could arise like opening streams and they would close at start_streaming scope end.
-    _sensors = _rs_dev.query_sensors();
-
     _device_sn = _rs_dev.get_info( RS2_CAMERA_INFO_SERIAL_NUMBER );
```

Comment:
I think that the comment that was here is important, it helps avoid a hard to detect bug.
Maybe move it to get_supported_streams (where we save the sensors now) or to _rs_sensors definition.

Reply:
What, about keeping the sensors?
OK, add back to get_supported_streams